exceptions.py

```diff
@@ -1,23 +1,74 @@
 # coding: utf-8
+from .resources import Message
```

Comment:
Why not import Message as ResinMessage similar to ResinException.

Personally I like more this import -- with the ResinBlah prefix -- but you should choose either one style and stick with it.